### PR TITLE
pdfstudioviewer: mark broken

### PR DESCRIPTION
**Pull request by tomcur**
###### Description of changes

Bad download hash. Probably unstable ([it's not the first time](https://github.com/NixOS/nixpkgs/commit/756c5ea52573f6ce302daf23f4428e92e539a702)).

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by tomcur** [comment]
(Ping @PhilippWoelfel: you were not tagged automatically as maintainer)

**Comment by PhilippWoelfel** [comment]
Please see https://github.com/NixOS/nixpkgs/pull/236917
As far as I can tell, the hash in this PR works.

File: pkgs/applications/misc/pdfstudio/default.nix [hunk 1]
```diff
@@ -40,6 +40,10 @@ in
       sha256 = "sha256-QXNsH1T+ItV3s9r8CnwgRUo1mhVbe8LkEun9gUmlVQg=";
     };
     jdk = jdk17;
+
+    # Bad hash, got sha256-afRhx9VCVRFUJoUnqs1bzF0yXpz3yEgLiFjMRB9xvsk=
```
Comment (doronbehar) [inline pkgs/applications/misc/pdfstudio/default.nix:44]: Nowadays upstream uses this link: https://download.qoppa.com/pdfstudioviewer/PDFStudioViewer_linux64.sh ...

**Review by doronbehar — APPROVED** [review]

**Comment by PhilippWoelfel** [comment]
Upstream is unstable. Link changes faster than I've been able to get pull request reviewed. Should this package be removed?

**Comment by doronbehar** [comment]
> Upstream is unstable. Link changes faster than I've been able to get pull request reviewed. Should this package be removed?

I was only iterating old packages. Feel free to submit such PR and ask for my review.

**Pull request merged**